flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskTest.java

```diff
@@ -879,6 +879,62 @@ public void testTaskConfig() throws Exception {
 		task.getExecutingThread().join();
 	}
 
+	@Test
+	public void testTerminationFutureCompletesOnNormalExecution() throws Exception {
+		final Task task = createTaskBuilder()
+			.setInvokable(InvokableBlockingWithTrigger.class)
+			.setTaskManagerActions(new NoOpTaskManagerActions())
```

Comment:
would it hurt to have `NoOpTaskManagerActions` as a default value for `TaskBuilder.taskManagerActions` instead of mock? The tests pass.

Reply:
Shouldn't hurt, let's do this in a follow-up.